src/DarkId.Papyrus.LanguageService/Projects/ProgramExtensions.cs

```diff
@@ -16,14 +17,24 @@ public static ProgramOptionsBuilder WithProject(this ProgramOptionsBuilder build
 
             builder.WithName(Path.GetFileNameWithoutExtension(projectInfo.ProjectFile))
                 .WithFlagsFileName(projectInfo.Project.Flags)
-                .WithSourceIncludes(projectInfo.Project.Imports.Where(i => !i.StartsWith("http", StringComparison.OrdinalIgnoreCase)).Reverse().Select(import => new SourceInclude()
-                {
-                    IsImport = true,
-                    Path = Path.GetFullPath(Path.Combine(projectFileDirectory, PathUtilities.Normalize(import)))
-                }));
+                .WithSourceIncludes(
+                    projectInfo.Project.Imports.Reverse().Select(import => 
+                    import.StartsWith("http")
+                    ? new SourceInclude(
+                                path: import,
+                                isImport: true,
+                                remotesInstallPath: builder.Build().RemotesInstallPath
+                    )
+                    : new SourceInclude()
+                    {
+                        IsImport = true,
+                        Path = Path.GetFullPath(Path.Combine(projectFileDirectory, PathUtilities.Normalize(import)))
+                    })
+                );
 
             if (projectInfo.Project.Folders != null && projectInfo.Project.Folders.Length > 0)
             {
+                // Huh??? does this just... include one folder? and that's it? huh????
```

Comment:
Yeah, totally don't remember.

Reply:
haha I planned to fix this too but forgot